PureLayout/PureLayout/ALView+PureLayout.m

```diff
@@ -250,6 +250,27 @@ - (NSLayoutConstraint *)autoPinEdgeToSuperviewMargin:(ALEdge)edge
     return [self autoPinEdgeToSuperviewMargin:edge relation:NSLayoutRelationEqual];
 }
 
+/**
+ Pins the given edge of the view to the corresponding margin of its superview with an inset.
+ 
+ @param edge The edge of this view to pin to the corresponding margin of its superview.
+ @param @param inset The amount to inset this view's edge from the corresponding margin of its superview edge.
+ @return The constraint added.
+ */
+- (NSLayoutConstraint *)autoPinEdgeToSuperviewMargin:(ALEdge)edge withInset:(CGFloat)inset
+{
+    self.translatesAutoresizingMaskIntoConstraints = NO;
+    ALView *superview = self.superview;
+    NSAssert(superview, @"View's superview must not be nil.\nView: %@", self);
+    if (edge == ALEdgeBottom || edge == ALEdgeRight || edge == ALEdgeTrailing) {
+        // The bottom, right, and trailing insets (and relations, if an inequality) are inverted to become offsets
+        inset = -inset;
+    }
+    ALMargin margin = [NSLayoutConstraint al_marginForEdge:edge];
+    
```

Comment:
Just to be consistent, can we remove redundant newline?

Reply:
Of course, thanks for catching.

Reply:
Thanks